rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -57,6 +64,48 @@
 
     private List<ModuleDescription> moduleDescriptions;
 
+    @Nullable
+    public byte[] coinbaseAddress() {
+        if (!minerServerEnabled()) {
```

Comment:
`localCoinbaseAccount` might be null if `!minerServerEnabled()` but also if `minerServerEnabled() && !config.hasPath(MINER_COINBASE_SECRET_CONFIG)`.

These are truly two different properties that might be accessed separately. I can't do one check at the moment. I think it would be different if I were to throw an exception if `!minerServerEnabled()`.

Reply:
That's right, I missed that case